forte/mrdsrg-spin-integrated/dsrg_mrpt2_grad/dsrg_mrpt2_deriv_solve_zvector.cc

```diff
@@ -970,6 +973,71 @@ void DSRG_MRPT2::z_vector_contraction(std::vector<double> & qk_vec, std::vector<
 
     y_ci("K") -= (Eref_ - Enuc_ - Efrzc_) * qk_ci("K");
 
+    /// Call the generalized sigma function to complete the contraction
+    for (const auto& pair: as_solver_->state_space_size_map()) {
+        const auto& state = pair.first;
+        std::map<std::string, double> block_factor;
+        block_factor["aa"] = 1.0;
+        block_factor["AA"] = 1.0;
+
+
+        auto antisym_1 = BTF_->build(CoreTensor, "antisymmetrized 1-body intermediate tensor", spin_cases({"aa"}));
```

Comment:
I think there is a misunderstanding of my slack message. There is nothing to be antisymmetrized for 1-body integrals.